client/src/pages/RootWorkspacePage/index.tsx

```diff
@@ -75,14 +75,14 @@ const RootWorkspace = ({ workspace }) => {
   return (
     <WorkspaceStyle>
       <Link to={`/workspaces/${workspace.id}`}>
-        <RootBlock block={question} />
+        <RootBlock block={question} availablePointers={[]} />
```

Comment:
could [] be the default value for the prop ``availablePointers`` instead, perhaps?